R/get_stats_utils.R

```diff
@@ -117,3 +117,13 @@ getHdi = function(v, mass) {
   r = HDInterval::hdi(v, credMass = mass)
   d = data.table(lower = r[1L], upper = r[2L])
   return(d)}
+
+
+getCondPairs = function(condLevels) {
+  if (!is.factor(condLevels)) condLevels = factor(condLevels)
+  condInt = as.integer(condLevels)
+  condMat = RcppAlgos::comboGrid(condInt, condInt, repetition = FALSE)
```

Comment:
Have you run devtools::check locally? We have a checklist in Notion now, right? I'm guessing this is related to the circle ci build failing.

Reply:
Yeah, I needed to add RcppAlgos to the DESCRIPTION and update some test expectations.